docs/decisions/0040-display-front-cover-in-preview-tab.md

```diff
@@ -0,0 +1,59 @@
+---
+parent: Decision Records
+nav_order: 40
+---
+
+# Display front cover for book citations in the Preview tab
+
+## Context and Problem Statement
+
+* Users have requested that the front covers of book citations are displayed in JabRef.
+* This is discussed on the [JabRef forum](https://discourse.jabref.org/t/display-cover-images-for-books/3647) and raised as a [feature request](https://github.com/JabRef/jabref/issues/10120).
+* We need to decide where the book cover should be placed.
+
+## Decision Drivers
+
+* It should not be obtrusive or distracting since the main use of JabRef is for articles not books.
+* It should not obstruct the view of existing GUI components, specifically the MainTable or the information in the EntryEditor's tabs.
+
+## Considered Options
+
+Place the book cover in:
+
+1. the existing SidePane
+2. a new SidePane
+3. the Preview panel of the EntryEditor
+4. a SplitPane next to the MainTable
+
+## Decision Outcome
+
+Chosen option: "3. The PreviewPanel of the EntryEditor".
+
+## Pros and Cons of the Options
+
+### 1. Existing SidePane
+
+![Image: Placement in SidePane](https://github.com/user-attachments/assets/7f704b0c-6f0c-4501-8167-4dc6202ca8f6)
```

Comment:
Please store the image in the repository, too.

Name it 0040-placement-sidepane.png. Do it for the others accordingly. (if it is jpg, use jpg as file extension)

Reply:
Should I put it in the docs/decisions folder?

Reply:
Yes. With the filename prefix `0040-*`, it will sorted next to the ADR.

Reply:
@damgam0288 I am still waiting for the images.

Reply:
I noticed that there's another ADR number 0040 - do you want me to change this ADR and the associated images to 0041 and then upload?

Reply:
Have uploaded them now

Reply:
@damgam0288 You also need to change all image links. That means, instead of github.com ..., the respective file should be referenced.

Reply:
sorry! I missed that. Have updated now.